index.src.html

```diff
@@ -505,6 +505,10 @@ <h3 id="headers">Additional CORS Headers</h3>
   The <dfn export http-header>`Access-Control-Allow-Private-Network`</dfn>
   indicates that a resource can be safely shared with external networks.
 
+  The <a http-header>`Private-Network-Access-Name`</a> attempt to provide users
+  a human friendly name in the [=private network access permission prompt=] (will
+  simpliy use the content permission prompt below).
```

Comment:
```suggestion
  The <a http-header>`Private-Network-Access-Name`</a> attempts to provide users
  a human friendly name in the [=private network access permission prompt=] (will
  simply use the content permission prompt below).
```

Reply:
I mean that in this document "permission prompt" specifically refers to "private network access permission prompt".

Reply:
I think instead of giving it a shorter name, it's probably clearer to just use the longer time consistently?